heroic-component/src/main/java/com/spotify/heroic/common/Feature.java

```diff
@@ -53,7 +53,12 @@
      * This will assert that there are data outside of the range queried for, which is a useful
      * feature when using a dashboarding system.
      */
-    SHIFT_RANGE("com.spotify.heroic.shift_range");
+    SHIFT_RANGE("com.spotify.heroic.shift_range"),
+
+    /**
+     * Enable feature to cause data to be fetched in slices.
```

Comment:
Changed to:
"Enable feature to cause data to be fetched in slices. This will cause data to be fetched and consumed by the aggregation framework in pieces avoiding having to load all data into memory before starting to consume it."
?